src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerTool.java

```diff
@@ -114,7 +115,8 @@ public DockerTool forNode(Node node, TaskListener log) throws IOException, Inter
         }
     }
 
-    @Extension public static class DescriptorImpl extends ToolDescriptor<DockerTool> {
+    @Extension @Symbol("docker")
```

Comment:
There is also a docker global environment variable in Docker Pipeline plugin. Is it safe?